src/main/client/src/components/@pages/QRScanResult.tsx

```diff
@@ -0,0 +1,37 @@
+import { Page } from '../@layout/Page'
+import React from 'react'
+import { ScanResponseDTO, ScanStatus } from 'types/dto/token'
+import { QRScanResultComponent } from 'components/@commons/QRScanResultComponent'
+import { useNavigate, useSearchParams } from 'react-router-dom'
+import { ButtonGroup, Button } from '@chakra-ui/react'
+import { FaArrowLeft, FaQrcode } from 'react-icons/fa'
+
+export const QRScanResult: React.FC = (props) => {
+  const [searchParams] = useSearchParams()
+  const server_response: ScanResponseDTO = {
+    title: searchParams.get('title') || undefined,
+    status: (searchParams.get('status') || ScanStatus.WRONG) as ScanStatus
+  }
+  const navigate = useNavigate()
+
+  const scanEventHandler = () => {
+    navigate('/qr/scan')
+  }
+  const backButtonHandler = () => {
+    navigate('/qr')
+  }
+
+  return (
+    <Page {...props}>
+      <QRScanResultComponent response={server_response} />
+      <ButtonGroup spacing="6">
```

Comment:
![Képernyőfotó 2022-01-18 - 9 11 01](https://user-images.githubusercontent.com/30603208/149898295-dffabee7-27e5-430b-a98e-2a260f9b3037.png)
Center it please!

Reply:
This what I was talking about when I said the heading size matches the page title. The icons and the layout will be dope though!